.pre-commit-config.yaml

```diff
@@ -0,0 +1,38 @@
+repos:
+  - repo: https://github.com/asottile/reorder_python_imports
+    rev: v3.0.1
+    hooks:
+      - id: reorder-python-imports
+        language_version: python3
+        args:
+          - --application-directories=.:wrapanapi
+  - repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+      - id: black
+        args: [--safe, --quiet, --line-length, "100"]
```

Comment:
--quiet ??

Reply:
Yep, limiting stderr output to the errors.

```
  -q, --quiet                     Don't emit non-error messages to stderr.
                                  Errors are still emitted; silence those with
                                  2>/dev/null.
```

Reply:
typically there is no need to silence black, in particualr when normally using it all the time, you never see any output